.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,25 @@
+Fixes #nnnn
 
 ---
 
-###### Reviewer checklist
+###### Review checklist
 
-- [ ] Test coverage: `stack test --coverage`
-- [ ] Public API documentation: `stack haddock`
+The author performs the actions on the checklist.
+The reviewer evaluates the work and checks the boxes as they are completed.
+
+-   [ ] **Summary**
+    Write a summary of the changes.
+    Explain what you did to fix the issue, and why you did it.
```

Comment:
Should this be mandatory, or can a clean commit history substitute it? My understanding from yesterday was that any of the two would be fine.